sdk/python/feast/staging/storage_client.py

```diff
@@ -262,14 +262,19 @@ def download_file(self, uri: ParseResult) -> IO[bytes]:
         Returns:
             TemporaryFile object
         """
-        url = uri.path
+        root = ""
```

Comment:
What problem does this change solve?

Reply:
It wasn't working if the path was like `file://user/folder1/text.parquet`. Since we are ignoring `netloc`, it tries to read `/folder1/text.parquet` and throws file not found.